build/mosquitto/patches/mux_epoll_c.patch

```diff
@@ -0,0 +1,10 @@
+--- a/src/mux_epoll.c	Wed Aug  4 12:43:36 2021
++++ b/src/mux_epoll.c	Wed Aug  4 12:44:11 2021
+@@ -28,6 +28,7 @@
+ #include <assert.h>
+ #ifndef WIN32
+ #ifdef WITH_EPOLL
++#include <signal.h>
+ #include <sys/epoll.h>
+ #define MAX_EVENTS 1000
+ #endif
```

Comment:
you seem to open two ifs but have only one endif ?

also can the assert.h be loaded in any environment ?

Reply:
I’m only inserting signal.h above says/poll.h, the other stuff is already in the file.

git does a poor job of showing patches.

Reply:
🙈 sorry